docs/tutorials/developer-journey/level-5/5.5-SNS-tutorial.md

```diff
@@ -0,0 +1,1097 @@
+
+All SNSs are hosted on the SNS subnet, which is exclusively used to host SNSs. This allows for simplified verification for end users, since end users can verify that an SNS is running on the SNS subnet, and therefore trust that the SNS's underlying code has been approved by the NNS community. 
+
+An SNS is comprised of the following canisters:
+
+- **SNS governance canister**: The governance canister defines who can participate in the SNS's governance decisions and automatically triggers the execution of all decisions. This canister stores the SNS proposals and a record of the neurons that define who the governance participants are. When a proposal is adopted, the governance system automatically triggers the execution of the proposal by calling a defined method. In most instances, the decision of the proposal is executed fully on-chain. 
+
+- **SNS ledger and archive canisters**: The ledger canister implements the ICRC-1 token standard to create a unique token for each SNS. To learn more about ICRC-1 tokens, refer to the previous module [4.2: ICRC-1 tokens](../level-4/4.2-icrc-tokens.md) where you deployed an ICRC-1 ledger then created an ICRC-1 token. This canister stores the SNS's ledger, which keeps track of how many accounts own SNS tokens, the amount owned by each account, and the history of transactions for the SNS's token. Since the canister has limited memory, to keep the ledger's full history, the ledger canister spawns archive canisters to store the ledger's block history. 
+
+- **SNS index canister**: The index canister keeps a record of which transactions are relevant for a given account. This provides the ability for wallets and other frontends to show all transactions for an account without requiring every frontend to implement this functionality themselves. When the NNS creates a new SNS, it will deploy the SNS's index canister as part of the deployment process. To deploy the index canister locally, follow the instructions found [here](/docs/current/developer-docs/integrations/icrc-1/icrc1-index-setup). 
+
+- **SNS root canister**: The root canister is responsible for upgrading the other SNS canisters and the dapp canisters that the SNS is responsible for governing. 
+
+- **SNS decentralization swap canister**: The decentralization swap canister, also called the swap canister, is the primary canister involved when an SNS is launched. In a decentralization swap, users provide ICP tokens that are swapped for the SNS's native token. For an SNS to be launched, the decentralization swap must collect a certain amount of initial funding. This canister collects that initial funding and facilitates the distribution of neurons (and therefore the voting power) to the participants of the decentralization swap, making the SNS's governance decentralized.
+
+### SNS asset canisters
```

Comment:
Do we need this section here?

Reply:
Resolving this since the SNS tutorial has been removed per my conversation with @LaraAS